README.md

```diff
@@ -27,6 +27,7 @@ This project is still under development. The following features with the check m
 - CMake >= 3.8.0
 - Rust >= 1.19.0
 - binutils >= 2.22
+- LLVM and Clang used by bindgen
```

Comment:
Can you clearly state the minimum version we need?

Reply:
I'm not sure, but LLVM and Clang with version 3.4.2 is fine. See https://github.com/pingcap/grpc-rs/issues/346#issuecomment-513187882.

Reply:
Can you add the version number to readme?

Reply:
Just use the version from CentOS7 repo.

Reply:
I just checked the docs from rust-bindgen, seems >3.9 is recommended. https://rust-lang.github.io/rust-bindgen/requirements.html

Reply:
updated